Frontend-Web/src/components/pages/error.tsx

```
@@ -0,0 +1,71 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import { Video, AlertCircle } from "lucide-react";
+import { useRouter } from "next/navigation";
+
+export default function Error() {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)